zipkin-server/src/test/java/zipkin2/server/internal/throttle/ThrottledStorageComponentTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package zipkin2.server.internal.throttle;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.composite.CompositeMeterRegistry;
+import static org.junit.Assert.assertSame;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import static org.mockito.Mockito.mock;
+import zipkin2.storage.StorageComponent;
+
+public class ThrottledStorageComponentTest {
```

Comment:
this probably should override toString with "Throttled("+delegate+")", so that the health check is easier to understand. It is currently like this and you can't tell which storage is in use:

```bash
$ curl -s localhost:9411/health
{"status":"UP","zipkin":{"status":"UP","details":{"ThrottledStorageComponent":{"status":"UP"}}}}
```